### ISSUE TITLE
Incompatible with shortcode plugin?

### ISSUE THREAD

**Issue by yoshikin** [Perlkonig/grav-plugin-table-importer#3]
This plugin seems to be incompatible with the Shortcode plugin. With Shortcode plugin enabled, the TableImporter shortcode is displayed as raw text `[TableImporter>data.csv]` instead of being converted to a table.

Is there any workaround for this?

**Comment by Perlkonig** [comment]
That's no good! I'm about to do a proper shortcode plugin and I'll see if I 
can find out what the problem is. Sorry about that.

**Comment by Perlkonig** [comment]
I'm afraid I can't replicate this. I installed `grav-plugin-shortcode-core` and enabled it and Table Importer is still working fine. Are you sure it's this plugin that's breaking it?

**Comment by SomeGuyFromNowhere** [comment]
Hello @Perlkonig 

I also tried using your table importer on a site theme that utilizes shortcodes in a plugin:  [Gravstrap](http://gravstrap.diblas.net) and it is not working. Instead of importing the table, the site shows the whole markdown. i need to deactivate the theme to make it run.

**Comment by Perlkonig** [comment]
I'll install Gravstrap on my dev server and see if I can find out what's happening.

**Comment by Perlkonig** [comment]
Well my dev server has shortcode-core and gravstrap plugins installed and enabled but [my dev demo page](https://perlkonig.com/dev/demos/table-importer) is working fine. Can someone post their page source code, including front matter? Thanks!

**Comment by Perlkonig** [comment]
I just released v2.0.0 which properly extends Shortcode Core. It now renders the HTML directly  as well. Please update the plugin and your shortcodes and let me know if you still have problems. Thanks!

**Comment by yoshikin** [comment]
The table importer plugin was working before the upgrade. I installed Shortcode Core, upgraded Table Importer and updated the shortcode on my page, but now I just see plain text.

I have cleared all caches and am just testing with a simple CSV file with the defaults (comma-separated, double quotes, etc.). I will try to do some testing to see where the problem is. I've tested with other shortcodes and they are working correctly.

I am using a customized version of the Learn2 theme, but I don't think this should affect shortcode behaviour.
